pkgs/development/python-modules/dataclasses-jsonschema/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, dataclasses
+, dateutil
+, jsonschema
+, typing-extensions
+, pytest
+, flake8
+, mypy
+, apispec
+, apispec-webframeworks
+, flask
+, pytest-runner
+, setuptools-scm
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "dataclasses-jsonschema";
+  version = "2.14.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "153v7vkkhqhiz7mzxbslbgqp2z7xcgh9hvg3lyppjaigi3pa22sg";
+  };
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace ", 'pytest-ordering'" ""
+  '';
+
+  nativeBuildInputs = [
+    setuptools-scm
+    pytest-runner
```

Comment:
```suggestion
```

Reply:
tests fail without this

Reply:
We can just remove the requirement from here https://github.com/s-knibbs/dataclasses-jsonschema/blob/master/setup.py#L29